docs/source/userguide_usage.rst

```diff
@@ -82,13 +82,9 @@ return `HIPSOLVER_STATUS_NOT_SUPPORTED` if called with the rocSOLVER backend.
   * :ref:`hipsolverDnXgesvdjSetMaxSweeps <compat_gesvdj_set_max_sweeps>`,
   * :ref:`hipsolverDnXgesvdjSetSortEig <compat_gesvdj_set_sort_eig>`,
   * :ref:`hipsolverDnXgesvdjSetTolerance <compat_gesvdj_set_tolerance>`,
-  * :ref:`hipsolverDnXgesvdjGetResidual <compat_gesvdj_get_residual>`, and
-  * :ref:`hipsolverDnXgesvdjGetSweeps <compat_gesvdj_get_sweeps>`.
-  * :ref:`hipsolverDnXsyevjSetMaxSweeps <compat_syevj_set_max_sweeps>`,
-  * :ref:`hipsolverDnXsyevjSetSortEig <compat_syevj_set_sort_eig>`,
-  * :ref:`hipsolverDnXsyevjSetTolerance <compat_syevj_set_tolerance>`,
-  * :ref:`hipsolverDnXsyevjGetResidual <compat_syevj_get_residual>`, and
-  * :ref:`hipsolverDnXsyevjGetSweeps <compat_syevj_get_sweeps>`.
+  * :ref:`hipsolverDnXgesvdjGetResidual <compat_gesvdj_get_residual>`,
+  * :ref:`hipsolverDnXgesvdjGetSweeps <compat_gesvdj_get_sweeps>`, and
+  * :ref:`hipsolverDnXsyevjSetSortEig <compat_syevj_set_sort_eig>`.
```

Comment:
Actually I was thinking about it, and I suggest we do add the non-sorting option to rocSOLVER. I think it could be useful. The other alternative method -iterative bisection (syevd)- also have a sorting/order option. So I think is good idea to have all the alternative methods providing a sorting option. 

I put together a quick PR in rocsolver to add this option. Please have a look and let me know your thoughts. 
https://github.com/ROCmSoftwarePlatform/rocSOLVER/pull/448